apps/inbound-mail/src/server/inbound-mail-queue.service.ts

```diff
@@ -12,11 +13,9 @@ export class QueueService {
       keyPrefix: process.env.REDIS_PREFIX ?? '',
     },
   };
-  public readonly bullMqService: BullMqService;
 
-  constructor() {
-    this.bullMqService = new BullMqService();
-    this.bullMqService.createQueue(this.QUEUE_NAME, {
+  constructor(public readonly bullMqService: BullMqService) {
+    this.bullMqService.createQueue(this.name, {
```

Comment:
As this app is not yet migrated to Nest.js we don't use the DI feature, ol' tradie instantiation.